src/tests/data/exampleStubPayload.json

```diff
@@ -22,19 +22,6 @@
             {
                 "value": "1960-02-02"
```

Comment:
I don't think we need need this cause we're not verifying DoB as part of BAV but happy to leave in

Reply:
Removed